extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcIdentityProvider.java

```diff
@@ -266,33 +367,57 @@ public void handle(AsyncResult<AccessToken> event) {
         }
     }
 
-    private static JsonObject getUserInfo(RoutingContext vertxContext, TokenAuthenticationRequest request,
+    private Uni<JsonObject> getUserInfoUni(RoutingContext vertxContext, TokenAuthenticationRequest request,
             TenantConfigContext resolvedContext) {
+        if (resolvedContext.oidcConfig.authentication.isUserInfoRequired()) {
+            if (BlockingOperationControl.isBlockingAllowed()) {
```

Comment:
Again, same issue.

Reply:
@cescoffier sure, hope the above comments, [this](https://github.com/quarkusio/quarkus/pull/13257/files#r522163580) and [this](https://github.com/quarkusio/quarkus/pull/13257/files#r522168791) ones, will clarify it :-)